core/pkg/opaprocessor/processorhandler.go

```diff
@@ -95,7 +98,7 @@ func (opap *OPAProcessor) Process(policies *cautils.Policies) error {
 
 	opap.loggerDoneScanning()
 
-	return errs
+	return nil
```

Comment:
TODO: Reconsider returning a list of errors (as current behavior)

Reply:
@dwertent I agree, this is an interesting point.

The current behavior is to never actually hydrate "errs", hence the simplification. Indeed, the "Process" function never returns an error.